app/models/story_distributions/episode_distribution.rb

```diff
@@ -13,7 +13,9 @@ def distribute!
   def add_episode_to_feeder(create_attributes = {})
     return unless url.blank?
     podcast = distribution.get_podcast
-    episode = podcast.episodes.post(episode_attributes.merge(create_attributes))
+    post_attributes = episode_attributes.merge(create_attributes)
```

Comment:
Is there a reason not to just put a default `url` in the `episode_attributes`?

It looks like the PodcastImport is always going to set `create_attributes[:url]` to at least nil.  So it should always override the beta-default in the merge call then, right?  (`{k: 'val'}.merge({k: nil}) == {k: nil}`)